stories/FileTree.stories.tsx

```diff
@@ -60,21 +64,26 @@ const version = createInternalVersion({
       },
     },
   },
-});
+};
 
 // We display an alert when a file has been selected.
 const onSelectFile = (path: string) => {
   // eslint-disable-next-line no-alert
   alert(`Selected file: ${path}`);
 };
 
-const render = ({
-  store = configureStore(),
-  ...moreProps
-}: { store?: Store } & Partial<FileTreeProps> = {}) => {
+const render = ({ ...moreProps }: Partial<FileTreeProps> = {}) => {
+  const store = configureStore();
+  store.dispatch(versionActions.loadVersionInfo({ version }));
+
+  const internalVersion = getVersionInfo(
+    store.getState().versions,
+    version.id,
+  ) as Version;
+
   const props: FileTreeProps = {
     onSelect: onSelectFile,
-    version,
+    versionId: internalVersion.id,
```

Comment:
Why do you need the full `internalVersion` object? Can't it just be `versionId: version.id`?

Reply:
Ah, you're right, we don't need the full `Version` object in the story anymore. Thanks!